ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/radius_server.schema.yml

```
@@ -0,0 +1,48 @@
+type: dict
+keys:
+  attribute_32_include_in_access_req:
+    type: dict
+    keys:
+      hostname:
+        type: bool
+      format:
+        type: str
+        description: Name of ACL
+  dynamic_authorisation:
```

Please make sure to follow wording of EOS CLI.
```suggestion
  dynamic_authorization:
```